recipes/numpy/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+cat > site.cfg <<'siteconfig'
+[DEFAULT]
+library_dirs = $PREFIX/lib
+include_dirs = $PREFIX/include
+
+[atlas]
+atlas_libs = openblas
+libraries = openblas
+
+[openblas]
+libraries = openblas
+library_dirs = $PREFIX/lib
+include_dirs = $PREFIX/include
+siteconfig
+
+$PYTHON setup.py config
+$PYTHON setup.py build -j $CPU_COUNT
```

Comment:
`CPU_COUNT` is a bit dubious with CI as they often misrepresent this figure. For instance, they might tell you how many cores there are on the node you are, but you may not be allowed to use that many. Normally, I just drop it entirely. In the case of `numpy`, it builds so fast I have never felt a need for this.